bin/i18n/i18n_metrics.rb

```diff
@@ -0,0 +1,44 @@
+require 'cdo/aws/metrics'
+require 'aws-sdk-ec2'
+require 'net/http'
+require 'socket'
+
+module I18n
+  module Metrics
+    class << self
+      attr_accessor :machine_id
+    end
+
+    def report_runtime(addtl_dimensions, value)
+      log_metric(:Runtime, addtl_dimensions, value)
+    end
+
+    def get_machine_id
+      begin
+        metadata_endpoint = 'http://169.254.169.254/latest/meta-data/'
+        return Net::HTTP.get(URI.parse(metadata_endpoint + 'instance-id'))
+      rescue
+        return "local_machine"
```

Comment:
There is no need to put `return` at the end of the method
```suggestion
        Net::HTTP.get(URI.parse(metadata_endpoint + 'instance-id'))
      rescue
        'local_machine'
```